docker/build_scripts/manylinux-check.py

```diff
@@ -1,6 +1,6 @@
 # Logic copied from PEP 513
 
```

Comment:
Should this file be named `docker/build_scripts/manylinux-check2010.py` ?

Reply:
Not sure it should be, that just another thing to change on the next update. I'd rather keep it that as it is.